ui/packages/consul-peerings/app/templates/dc/peers/show/index.hbs

```diff
@@ -3,6 +3,6 @@
   SPDX-License-Identifier: BUSL-1.1
 }}
 
-<Route @name={{routeName}} as |route|>
-  {{did-insert this.transitionToImported}}
+<Route @name={{this.routeName}} as |route|>
+  {{did-insert-helper this.transitionToImported}}
```

Comment:
as far as I know `did-insert-helper` set of helpers are deprecated in favour of `did-insert` etc., maybe be out of scope of this PR to refactor but it as well just keep `did-insert`